ml-agents/mlagents/trainers/policy/tf_policy.py

```diff
@@ -26,6 +31,8 @@
 # determines compatibility with inference in Barracuda.
 MODEL_FORMAT_VERSION = 2
 
+EPSILON = 1e-6  # Small value to avoid divide by zero
```

Comment:
We create a bunch of these EPSILON throughout the code and they are not always the same value. Maybe out of scope for this PR, but we should have a global EPSILON somewhere.

Reply:
Totally agree. Where should we put the global one? I imagine we'd just import it like `from mlagents.trainers.epsilon import EPSILON` or something like that.

Reply:
`constants.py`?